### DISCUSSION TITLE
Can I use this to throttle people?

### DISCUSSION THREAD

**Discussion by caspervanpomeren** [thumbmarkjs/thumbmarkjs#57] [Q&A] [answered]
Does this library allow me to accurately distinguish between people that go to my website? For example, I first wanted to use IP addresses, but those can be used by multiple people (or even hundreds). Can this also happen with thumbmark? So that a lot of people share one thumbmark? Or can I safely assume almost everyone will get a unique code?

**Comment by Joe12387** [comment, marked as answer]
The short answer is no, there will be many browsers that have the same ID with this library.
